CHANGELOG.md

```diff
@@ -28,6 +28,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Adds CAPABILITY_REQ global into the zome context. This makes the request available during all zome calls and is important for doing validation of provenance for a zome call that wants to create a capability grant. [#1273](https://github.com/holochain/holochain-rust/pull/1273)
```

Comment:
@zippy can you reword this?
This makes no sense to me...  I'll break down what I don't understand specifically if that helps...
What's "global into the zome context"?
"the request" which request?
"a capability grant" What's a capability grant, from the perspective of someone who's never heard of it before?